### ISSUE TITLE
Parallelism and multiple sessions

### ISSUE THREAD

**Issue by Volodymyr128** [spark-jobserver/spark-jobserver#815]
Is it possible  to run in parallel multiple jobs with different # of cores and RAM per executor?

I'm using Spark standalone cluster manager. As you know, to change amount of RAM per executor I have to start another Spark session (another JVM). Is that possible to start 3-5 spark sessions from Spark Job Server in parallel?

**Comment by velvia** [comment]
Yes, i don't see why not as long as the cluster has enough resources.  Note that starting another JVM is not strictly necessary. You can start multiple contexts in SJS with different parameters without a separate JVM per context.  

-Evan
"Never doubt that a small group of thoughtful, committed citizens can change the world" - M. Mead
"We can do no great things, just small things with great love." -mother Teresa

**Comment by Volodymyr128** [comment]
@velvia thank u for quick reply! Do you have any example of how create multiple SJS contexts with Java API?

**Comment by velvia** [comment]
The contexts are created using the REST HTTP API, not from within a job!  You have to create a context first before you can submit a job.

**Comment by Volodymyr128** [comment]
What does it mean [here](https://github.com/spark-jobserver/spark-jobserver#features):

> Separate JVM per SparkContext for isolation (EXPERIMENTAL)

@velvia Do I need to launch multiple JVMs on spark job server?

**Comment by noorul** [comment]
@Volodymyr128 You just need to enable jvm per context feature.

**Comment by Volodymyr128** [comment]
@noorul thank you for support! Could you please take a look at two more questions about deployment? If you want - I can create a separate issue

From [here](https://github.com/spark-jobserver/spark-jobserver#creating-a-project-manually-assuming-that-you-already-have-sbt-project-structure) 
> To create a job that can be submitted through the job server, the job must implement the SparkJob trait

I don't need to write any line of Scala code, If I want to use [Java Spark Job Server Client](https://github.com/bluebreezecf/SparkJobServerClient), right?

Where I can find all necessary templates and sh files for manual deployment described [here](https://github.com/spark-jobserver/spark-jobserver#manual-steps)? What is the easiest way to deploy job server to cluster  with Standalone manager if I use  [Java Spark Job Server Client](https://github.com/bluebreezecf/SparkJobServerClient)?

**Comment by noorul** [comment]
You need to follow these steps to get the spark jobserver jar built https://github.com/spark-jobserver/spark-jobserver#getting-started-with-spark-job-server

**Comment by Volodymyr128** [comment]
Thank you for support!

**Comment by velvia** [comment]
You’re welcome  :)
